<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListEditWidget</class>
 <widget class="QWidget" name="ListEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Список</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Добавить</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+I</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="deleteButton">
     <property name="text">
      <string>Удалить</string>
     </property>
     <property name="shortcut">
      <string>Del</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
